test/user.test.js

```diff
@@ -590,6 +603,40 @@ describe('User', function() {
         });
       });
     });
+
+    it('Should be able to use mixed-case pattern (email case-sensitivity on)', function(done) {
```

Comment:
We avoid using `should` in test names (at least in the new code), see http://loopback.io/doc/en/contrib/style-guide-es6.html#test-naming

```js
it('supports email searches with "like" (email case-sensitivity on)', function(done) {
  // ...
});
```